packages/terra-application-links/src/tabs/ApplicationTabs.module.scss

```
@@ -1,3 +1,8 @@
+
+// Themes
+@import './clinical-lowlight-theme/ApplicationLinks.module';
+@import './orion-fusion-theme/ApplicationLinks.module';
+
```

These should be importing from 
```suggestion
@import './clinical-lowlight-theme/ApplicationTabs.module';
@import './orion-fusion-theme/ApplicationTabs.module';

```

---

Done 1fa7f41